app/Actions/Diagnostics/Pipes/Checks/AppUrlMatchCheck.php

```diff
@@ -47,38 +50,54 @@ public function handle(array &$data, \Closure $next): array
 				$censored_bad,
 				$censored_current,
 			);
+			// @codeCoverageIgnoreEnd
 		}
 
 		if ($bad !== '') {
+			// @codeCoverageIgnoreStart
 			$data[] = sprintf(
 				'Warning: APP_URL (%s) contains a sub-path (%s).',
 				$censored_app_url,
 				$censored_bad
 			);
 			$data[] = self::INVISIBLE_WARNING . 'This may impact your WebAuthn authentication.';
+			// @codeCoverageIgnoreEnd
 		}
 
 		if (!$this->checkUrlMatchCurrentHost()) {
+			// @codeCoverageIgnoreStart
 			$data[] = sprintf(
 				'Error: APP_URL (%s) does not match the current url (%s).',
 				$censored_app_url,
 				$censored_current,
 			);
 			$data[] = self::INVISIBLE_ERROR . 'This will break WebAuthn authentication.';
+			// @codeCoverageIgnoreEnd
 		}
 
 		$config_url_imgage = config('filesystems.disks.images.url');
 		if ($config_url_imgage === '') {
+			// @codeCoverageIgnoreStart
 			$data[] = 'Error: LYCHEE_UPLOADS_URL is set and empty. This will prevent images to be displayed. Remove the line from your .env';
+			// @codeCoverageIgnoreEnd
 		}
 
-		if (($config_url . $dir_url . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
+		if (!str_starts_with($config_url_imgage, '/') && !str_starts_with($config_url_imgage, 'http')) {
+			// @codeCoverageIgnoreStart
+			$data[] = 'Error: LYCHEE_UPLOADS_URL is set but start with neither a / or http.';
+			$data[] = self::INVISIBLE_ERROR . 'This will prevent images to be displayed. Remove the line from your .env';
```

Comment:
```suggestion
			$data[] = self::INVISIBLE_ERROR . 'This will prevent images from being displayed. Remove the line from your .env';
```

I swear it's the third time I'm marking this?! :smile:

Reply:
Yeah same. Weird.